src/Integration.Vsix.UnitTests/InfoBar/InfoBarManagerTests.cs

```diff
@@ -327,6 +383,16 @@ private static ConfigurableVsInfoBarHost RegisterFrameInfoBarHost(ConfigurableVs
             return host;
         }
 
+        private static void RegisterMainWindowHostWithShell(Mock<IServiceProvider> serviceProvider, IVsInfoBarHost host)
```

Comment:
I tweaked this helper method so the caller passes in the host to return. Needed so that one of the new tests can simulate the service returning a null host.